tests/robot-tests/scripts/create_snapshots.py

```diff
@@ -265,13 +266,6 @@ def write_snapshots_to_file(self):
         required=False,
     )
 
-    ap.add_argument(
-        "--validate",
-        dest="validate",
-        action="store_true",
-        help="Compares the latest snapshots to the existing versions and does not write any snapshot files",
-    )
-
```

Comment:
I think it was me that added this option because I thought it's useful for running locally.
Couldn't we leave this in but just remove `--validate` from the run command like you've already done?

Reply:
Ah right ok, yep sounds good

Reply:
Ahh, I think I see the problem you were trying to resolve now. The action needs to both validate and write the files.